packages/udaru-core/package.json

```diff
@@ -44,6 +44,7 @@
     "depcheck": "npx depcheck",
     "pg:init": "node ./database/init.js && npm run pg:migrate",
     "pg:init-test-db": "npm run pg:init && npm run pg:load-test-data",
+    "pg:init-volume-db": "npm run pg:init && node ./database/loadVolumeData.js",
```

Comment:
we seem to have this in `udaru-core` and `udaru-hapi-server`, is there a good reason it's in core?

Reply:
possibly not, it was just in there so I left it... it could be useful for us to have some tests with the volume data set but there are none at the moment so it could probably be moved out altogether... currently it's only used for the bench tests but I've found it handy for testing manually

Reply:
Let's remove from core and keep it all in server, thanks